src/context/useVillagers.jsx

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react';
```

Comment:
this isn't actually context. As such, I like to make custom hooks in a hooks/ folder.